pkg/sbproxy/reconcile/reconcile_brokers_test.go

```diff
@@ -373,6 +374,46 @@ var _ = Describe("Reconcile brokers", func() {
 				}
 			},
 		}),
+
+		Entry("When broker is registered in the platform, but not yet known to the proxy, it should be updated", testCase{
+			stubs: func() {
+				stubPlatformOpsToSucceed()
+				fakePlatformBrokerClient.UpdateBrokerReturns(&platform.ServiceBroker{
```

Comment:
could you (similarly like other tests use just a variable with a meaningful name in the stubs (keep it readable)